contents/blog/early-stage-analytics.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Early-stage startups need analytics, here’s why (and what matters most)"
```

Comment:
I quite like 'The 80/20 of early-stage startup analytics' for the actual headline... thoughts? It's nice and punchy.

Reply:
I like it more too